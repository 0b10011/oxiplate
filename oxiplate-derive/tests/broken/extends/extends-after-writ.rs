use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline(r#"{{ title }}{% extends "multiple-blocks.html.oxip" %}"#)]
struct Writ {
    title: &'static str,
}

fn main() {
    print!("{}", Writ {
        title: "Double extends",
    });
}
