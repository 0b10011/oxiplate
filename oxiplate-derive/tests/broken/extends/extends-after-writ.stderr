error: proc-macro derive panicked
 --> tests/broken/extends/extends-after-writ.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: not yet implemented: Can't extend if already adding content

error[E0277]: `Writ` doesn't implement `std::fmt::Display`
  --> tests/broken/extends/extends-after-writ.rs:10:18
   |
10 |       print!("{}", Writ {
   |  _____________--___^
   | |             |
   | |             required by this formatting parameter
11 | |         title: "Double extends",
12 | |     });
   | |_____^ `Writ` cannot be formatted with the default formatter
   |
   = help: the trait `std::fmt::Display` is not implemented for `Writ`
   = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
   = note: this error originates in the macro `$crate::format_args` which comes from the expansion of the macro `print` (in Nightly builds, run with -Z macro-backtrace for more info)
